src/main/resources/db/migration/V20191025141243__users_studies_extended.sql

```diff
@@ -0,0 +1,34 @@
+CREATE OR REPLACE VIEW users_studies_extended AS
+SELECT DISTINCT ON (data.study_id, data.user_id, data.role, data.status, data.owned_data_source_id)
```

Comment:
Bugfix itself is adding owner_data_source_id to the DISTINCT expression.
Not having it there cause merging different records into one

Reply:
ok